docs/api-docs/catalog/price-list-overview.md

```diff
@@ -40,17 +41,17 @@ The following OAuth Scopes are required:
 ```
 
 
-- `PriceRecordBatch`: A way to update several `Price Records` in a `Price List` at once. Using this bulk upsert endpoint, you can upsert up to 1000 Price Records in a single API call.
+- `PriceRecordBatch` -- A way to update several `Price Records` in a `Price List` at once. Using this bulk upsert endpoint, you can upsert up to 1000 Price Records in a single API call.
 
-- `Currency`:  A `Price List` can contain records for multiple currencies. At this time, *only price records that match the store's default currency will be used to determine storefront and in-cart prices.* Although BigCommerce supports a storefront currency selection, this is not currently integrated with Price Lists and will merely convert prices from the store's default currency for display convenience.
+- `Currency` --  A `Price List` can contain records for multiple currencies. *Only price records that match the store's default currency determine storefront and in-cart prices.* Although BigCommerce supports a storefront currency selection, this is not currently integrated with Price Lists and will merely convert prices from the store's default currency for display convenience.

```

Comment:
We will use price records for transactional currencies.  If the merchant's default us USD and they have EUR set up as a transactional currency in the control panel, when a shopper selects EUR we will look for EUR price records.  If those don't exist, we fall back to catalog pricing and convert it.  

If the alternate currencies are display only, we'll read the store default currency price records and convert for display.